.github/workflows/codecov.yml

```diff
@@ -0,0 +1,45 @@
+name: Run Codecov on PR open or change
+on: pull_request
+jobs:
+  codecov:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Check out this repository code
+        uses: actions/checkout@v3
+        with:
+          path: empower
+          fetch-depth: 0
+
+      - name: Check out `empower-config` to get env-config
+        uses: actions/checkout@v3
+        with:
+          repository: sentry-demos/empower-config
+          path: empower-config
+          token: ${{ secrets.KOSTY_PERSONAL_ACCESS_TOKEN_FOR_SYNC_DEPLOY_FORK }}
```

Comment:
@realkosty is this `Check out empower-config to get env-config` needed to run tests or no? I wasn't clear, let me know if I should delete it.

Reply:
Yes, it should not be needed if tests are written properly as true unit tests (e.g. DB connection mocked instead of actually attempting to create a real DB connection with empty host/username/password from missing env variable).

My question is why a separate workflow instead of `auto-deploy.yml`?

Reply:
Oh! this is for comment in PR, sorry never mind
codecov upload in `auto-deploy` is for Sentry code context integration.

Reply:
Thanks, will remove it then 👍 

> My question is why a separate workflow instead of `auto-deploy.yml`?

Eventually we want the ability to start a demo from the PR - showcase Codecov comment, then Sentry-bot comment about an error being detected (second part will come later). For this reason we need this workflow to trigger upon PR open/modification within the Empower repo.

The existing auto-deploy workflow currently triggers [upon push to `master`](https://github.com/sentry-demos/empower/blob/master/.github/workflows/auto-deploy.yml#L5-L7). Beyond the trigger being different from what we want, the auto-deploy workflow does a bunch of other things like deployment that we don't want to run when a new PR opens (including deploying).

LMK what you think!